src/agent/index.ts

```diff
@@ -61,17 +70,18 @@ export const didProviders = {
 }
 
 const dbConnection = getDbConnection(DB_CONNECTION_NAME)
+const privateKeyStore: PrivateKeyStore = new PrivateKeyStore(dbConnection, new SecretBox(DB_ENCRYPTION_KEY))
 
 const agent = createAgent<
-  IDIDManager & IKeyManager & IDataStore & IDataStoreORM & IResolver & IDidAuthSiopOpAuthenticator & IConnectionManager
+  IDIDManager & IKeyManager & IDataStore & IDataStoreORM & IResolver & IDidAuthSiopOpAuthenticator & IConnectionManager & ICredentialHandlerLDLocal
```

Comment:
I am missing ICredentialssuer (probably you are using our VC hanlder ld local directly to issue credentials